### ISSUE TITLE
Getting SSL/TLS Error while cloning repository

### ISSUE THREAD

**Issue by nikhilnyl** [desktop/desktop#14186]
Hi Team,

When I am trying to pull code from github using **github application**,I am getting SSL/TLS handshake exception.
I am able to access the code from website but not able to clone it using application. Even I checked my certs are placed properly and no issue is there. please help

**Comment by steveward** [comment]
Thanks for the report @nikhilnyl. Can you share the output of the error you are seeing, along with a log file from GitHub Desktop?

To access the log files go to the file menu in GitHub Desktop and select `Help` > `Show Logs`.
The log files are created daily -- please upload a log file as an attachment from a day where you experienced the issue.

**Comment by Arndress** [comment]
I am experiencing the same issue as of yesterday, much as I did last year with #12743. Interestingly and perhaps coincidentally, what myself and the other affected user had in common last time is that we both were running Windows 7. I am still running Windows 7.

I have attached today's log file: [2022-05-28.desktop.production.log](https://github.com/desktop/desktop/files/8791475/2022-05-28.desktop.production.log)
